.github/workflows/ci.yml

```diff
@@ -26,11 +28,14 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 18
+          cache: npm
       - run: npm ci
       - run: npm run typecheck
       - run: npm run lint
+      - run: npm run build:embedded # check that build works
```

Comment:
Added build check to prevent case, when release is created, but build unsuccessful and artifacts are not uploaded